src/test/java/com/jalasoft/selenium/daniel/jauregui/movies/ChildrenTest.java

```diff
@@ -0,0 +1,79 @@
+package com.jalasoft.selenium.daniel.jauregui.movies;
+
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * ChildrenTest.
+ * @author Daniel Jauregui
+ */
+public class ChildrenTest {
+    private MovieAbs instance;
+    private static final double DELTA = 0.0;
+    /**
+     * Initialize a object.
+     */
+    @Before
+    public void setUp() {
+        // Given
+        instance = new Children("Big Hero", 0);
+    }
+
+    /**
+     * Test of testCalculateRateWithRentedDaysMajorThan3.
+     */
+    @Test
+    public void testCalculateRateWithRentedDaysMajorThan3() {
+        //When
+        final int rentedDays = 32;
+        final double expResult = 45;
+        double result = instance.calculateRate(rentedDays);
+        //Then
+        System.out.println("testCalculateRateWithRentedDaysMajorThan3");
+        assertEquals(expResult, result, DELTA);
+    }
+
+    /**
+     * Test of calculattestCalculateRateWithRentedDaysLessThan3.
+     */
+    @Test
+    public void testCalculateRateWithRentedDaysLessThan3() {
+        //When
+        final int rentedDays = 1;
+        final double expResult = 1.5;
```

Comment:
expResult variable should be in the //Then section

Remember the variables should be declared in the near scope